x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/result_settings_text_fields_body.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiTableBody,
+  EuiTableRow,
+  EuiTableRowCell,
+  EuiTableRowCellCheckbox,
+  EuiCheckbox,
+} from '@elastic/eui';
+
+import { ResultSettingsLogic } from '../result_settings_logic';
+import { FieldResultSetting } from '../types';
+
+import { ResultSettingsFieldNumber } from './result_settings_field_number';
+
+export const ResultSettingsTextFieldsBody: React.FC = () => {
+  const { textResultFields } = useValues(ResultSettingsLogic);
+  const {
+    toggleRawForField,
+    updateRawSizeForField,
+    clearRawSizeForField,
+    toggleSnippetForField,
+    updateSnippetSizeForField,
+    clearSnippetSizeForField,
+    toggleSnippetFallbackForField,
+  } = useActions(ResultSettingsLogic);
+
+  const resultSettingsArray: Array<[string, Partial<FieldResultSetting>]> = useMemo(() => {
+    return Object.entries(textResultFields).sort(([aFieldName], [bFieldName]) =>
+      aFieldName > bFieldName ? 1 : -1
+    );
+  }, [textResultFields]);
+
+  return (
+    <EuiTableBody>
+      {resultSettingsArray.map(([fieldName, fieldSettings]) => (
+        <EuiTableRow key={fieldName}>
+          <EuiTableRowCell data-test-subj="ResultSettingFieldName">{fieldName}</EuiTableRowCell>
```

Comment:
Minor typography suggestion here to get this UI more on par with the standalone

```suggestion
          <EuiTableRowCell data-test-subj="ResultSettingFieldName">
            <code>{fieldName}</code>
          </EuiTableRowCell>
```